keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlArtifactRepository.kt

```diff
@@ -1210,7 +1211,8 @@ class SqlArtifactRepository(
     deliveryConfig: DeliveryConfig,
     environmentName: String,
     artifactReference: String,
-    version: String
+    version: String,
+    verifications: List<ArtifactVersionEnvironmentVerificationSummary>
```

Comment:
Instead of passing this as a parameter, I think it might be better to just add that via `copy()` to the return object in the caller, since this method is not doing anything with the parameter besides sticking it in there.

Reply:
That's a good idea. Will do.